test/translate-test.js

```diff
@@ -0,0 +1,39 @@
+const chai = require('chai');
```

Comment:
```suggestion
/* eslint-disable no-unused-expressions */
const chai = require('chai');
```
Consider adding this to the translate-test.js file, since I am seeing the following esline warning throughout the `expect` statements:

> Expected an assignment or function call and instead saw an expression. eslint[no-unused-expressions](https://eslint.org/docs/rules/no-unused-expressions)